### ISSUE TITLE
Web Standard Report

### ISSUE THREAD

**Issue by dbeauvai** [hres/pmp#71]
@OttawaOn - Hi Ching, Please find attached the web standard report that was sent to us by the web standard group.  They did an assessment of the form and they have made recommendation.  This document will help you identify the changes that has to be done to the application/form.
[DHPR - Web Standards and Web Renewal Compliance Report.docx](https://github.com/hres/pmp/files/97510/DHPR.-.Web.Standards.and.Web.Renewal.Compliance.Report.docx)

**Comment by MartinBernard** [comment]
@dbeauvai - info on web standards as well as the report can be found under #39 .

**Comment by OttawaOn** [comment]
Hi Diane,
I received the web standard document. I am working on it.
Thanks!
Ching

**Comment by OttawaOn** [comment]
Deare Diane,
Thanks for the email!
I received web standard document.
I am working on it.
Thanks!
Ching

**Comment by OttawaOn** [comment]
Dear Diane,
How are you!
Last Friday I met Mona (my recruiter form Calian company), we had a very
nice talk in the

cafeteria; I went back to my computer, the computer updated some software
I finished converting a table in cover page and I edited Cover page and
Part one(delete some unused code). I put label tag and attributes like
'lang=en' inside them. I will finish putting them (Labels) on Part two and
Part three on next week, also make more changes.

I got the email message from Peterburge by Mike.

Ok, Nice weekends!

Ching
